modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -2813,6 +2949,131 @@ Fields absent from the 'k8s.v1.cni.cncf.io/network-status' annotation must not b
 | `string`
 | `name` should match the network name as visible in the pods annotation 'k8s.v1.cni.cncf.io/network-status'.
 
+|===
+== .spec.processor.deduper
+Description::
++
+--
+`deduper` allows to sample or drop flows identified as duplicates, in order to save on resource usage.
+IMPORTANT: This feature is available as a Developer Preview.
+--
+
+Type::
+  `object`
+
+
+
+
+[cols="1,1,1",options="header"]
+|===
+| Property | Type | Description
+
+| `mode`
+| `string`
+| Set the Processor de-duplication mode. It comes in addition to the Agent-based deduplication because the Agent cannot de-duplicate same flows reported from different nodes. +
+
+- Use `Drop` to drop every flow considered as duplicates, allowing saving more on resource usage but potentially loosing some information such as the network interfaces used from peer, or network events. +
+
+- Use `Sample` to randomly keep only 1 flow on 50 (by default) among the ones considered as duplicates. This is a compromise between dropping every duplicates or keeping every duplicates. This sampling action comes in addition to the Agent-based sampling. If both Agent and Processor sampling are 50, the combined sampling is 1:2500. +
```

Comment:
```suggestion
- Use `Sample` to randomly keep only one flow on 50, by default, among the ones considered as duplicates. This is a compromise between dropping every duplicate or keeping every duplicate. This sampling action comes in addition to the Agent-based sampling. If both Agent and Processor sampling are 50, the combined sampling is 1:2500. +
```

Reply:
Do the two instances of 50 need markup as values?
```suggestion
- Use `Sample` to randomly keep only 1 flow on `50` (by default) among the ones considered as duplicates. This is a compromise between dropping every duplicates or keeping every duplicates. This sampling action comes in addition to the Agent-based sampling. If both Agent and Processor sampling are `50`, the combined sampling is 1:2500. +
```

Reply:
I would say only the second is a direct reference to the configured value, the first is more an indirect reference.